sysl2/sysl/seqs.go

```diff
@@ -76,28 +76,83 @@ func escapeWordBoundary(src string) string {
 	return val
 }
 
-func DoConstructSequenceDiagrams(
+func DoConstructSequenceDiagramsWithParams(
 	rootModel, endpointFormat, appFormat, title, output, modules string,
 	endpoints, apps []string,
 	blackboxes [][]string,
 	group string,
+	loglevel string,
+	verbose bool,
 ) (map[string]string, error) {
+	cmdContextParamSeqgen := &CmdContextParamSeqgen{
+		root:           &rootModel,
+		endpointFormat: &endpointFormat,
+		appFormat:      &appFormat,
+		title:          &title,
+		output:         &output,
+		modulesFlag:    &modules,
+		endpointsFlag:  &endpoints,
+		appsFlag:       &apps,
+		blackboxes:     &blackboxes,
+		loglevel:       &loglevel,
+		group:          &group,
+		verbose:        &verbose,
+	}
+	return DoConstructSequenceDiagrams(cmdContextParamSeqgen)
+}
+
+func DoConstructSequenceDiagrams(cmdContextParam *CmdContextParamSeqgen) (map[string]string, error) {
+	var blackboxes [][]string
+
+	log.Debugf("root: %s\n", *cmdContextParam.root)
+	log.Debugf("endpoints: %v\n", cmdContextParam.endpointsFlag)
+	log.Debugf("app: %v\n", cmdContextParam.appsFlag)
+	log.Debugf("endpoint_format: %s\n", *cmdContextParam.endpointFormat)
+	log.Debugf("app_format: %s\n", *cmdContextParam.appFormat)
+	log.Debugf("title: %s\n", *cmdContextParam.title)
+	log.Debugf("modules: %s\n", *cmdContextParam.modulesFlag)
+	log.Debugf("output: %s\n", *cmdContextParam.output)
+	log.Debugf("loglevel: %s\n", *cmdContextParam.loglevel)
+
+	if cmdContextParam.plantuml == nil {
+		plantuml := os.Getenv("SYSL_PLANTUML")
+		cmdContextParam.plantuml = &plantuml
+		if *cmdContextParam.plantuml == "" {
+			*cmdContextParam.plantuml = "http://localhost:8080/plantuml"
```

Comment:
I feel that a warning log to inform local plantuml server is being used will be helpful

Reply:
This would be too noisy. Using localhost is generally safer than the alternative anyway, and if no local server is running, the inevitable error message should make it clear.